extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -1035,6 +1035,7 @@ export class AmpConsent extends AMP.BaseElement {
 
     // Set up the __tcfApiLocator window to signal postMessage support
     const iframe = this.element.ownerDocument.createElement('iframe');
+    iframe.setAttribute('title', this.element.title || 'AMP Consent');
```

Comment:
Is "Consent Banner" better? AMP does not provide information to the user.

Reply:
Makes sense, I have updated the default value with the suggestion